rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -123,16 +160,73 @@
           "type": "string",
           "pattern": "^https://ils.rero.ch/api/item_types/.+?$",
           "form": {
+            "templateOptions": {
+              "hideLabel": true
+            },
             "remoteOptions": {
               "type": "item_types"
             }
           }
         }
       },
       "form": {
+        "templateOptions": {
+          "cssClass": "editor-title"
+        },
         "hideExpression": "field.parent.model.type === 'issue'"
       }
     },
+    "temporary_item_type": {
+      "title": "Temporary circulation category of the item",
+      "description": "Apply a temporary item circulation category for this item. If specified, this circulation category overrides the default category.",
+      "type": "object",
+      "propertiesOrder": [
+        "$ref",
+        "end_date"
+      ],
+      "required": [
+        "$ref"
+      ],
+      "properties": {
+        "$ref": {
+          "title": "Item Type URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/item_types/.+?$",
+          "form": {
+            "templateOptions": {
+              "hideLabel": true
+            },
+            "remoteOptions": {
+              "type": "item_types"
+            }
+          }
+        },
+        "end_date": {
+          "title": "End date",
+          "description": "If set, specifies the date when the temporary circulation category will be disabled.",
```

Comment:
`will be removed`?

Reply:
This description seems important ? @iGormilhit helps me to get a correct sentence. @iGormilhit, can you confirm my feeling ?

Reply:
[Un]fortunately, @BadrAly is right. Sorry. :slightly_smiling_face: